### DISCUSSION TITLE
mapping command button

### DISCUSSION THREAD

**Discussion by ratatatat12** [noah-nuebling/mac-mouse-fix#467] [Questions]
I'm trying to map the mac 'cmd' button onto click 5 of my m510 mouse on my mac. Mac mouse fix doesn't allow just the cmd button to be assigned so I tried mapping cmd+` and cmd+enter. This worked for a couple of clicks and now has stopped working. Is there another work around for this?

**Comment by noah-nuebling** [comment]
Hey @ratatatat12, thanks for your feedback!

Mapping to just modifier keys isn't supported unfortunately.

That's because it would need a very different UI and backend than the currently implemented "Keyboard Shortcut..." feature. And it would also only work in specific configurations. E.g. When you assign a keyboard modifier to "Click"ing a button it could work but only if there is no "Hold" or "Double Click" set up. As soon as you set up "Hold" or "Double Click", the keyboard modifier assigned to "Click" would stop working. So it could sort of work from a technical standpoint but it would make for a very confusing user experience.

But if you explain to me why you're trying to assign the Command key to a mouse button maybe there something else I can implement in MMF to help.

**Reply by ratatatat12** [reply]
Thanks for the reply. That makes sense. When I'm using a web browser, I like to open links in new tabs. Currently I need one hand on the cmd keyboard key and one clicking the mouse. I'm hoping to find a way to do this with just the mouse without having to right click and select 'open link in new tab'. On my windows PC I'm able to map the control key to the side button of the mouse which allows this, but unfortunately not on my mac.

**Reply by noah-nuebling** [reply]
Middle click should have the same effect if I understand correctly. 

You can delete all actions for the Middle Button in MMF and then just use it like that or use the "Middle Click" option in MMF if you also want to use Hold, Click and Drag, etc. with your middle button.

Let me know if you have more questions!

**Reply by to-turion** [reply]
I'm a little confused about this, myself. I had the middle button on my Unimouse mapped to ⌘+click for ages, but Contour's new driver for Sonoma turned it into chaos machine, so I've been looking for alternatives. Since starting to use Mac Mouse Fix instead, I've noticed that the middle button works _very_ differently depending on the context. It's sometimes equivalent to ⌘+click, sometimes not — and not always consistently within the same apps. In one context, it multi-selects. In another, it closes things.

I'm sure I'll get used to it eventually, but I would be more than happy to give up my "hold" and "double click" actions to turn my middle button back into ⌘+click. Right now, all my hold actions are set to the same things as their equivalent single clicks. I think I'm pretty speedy, but they kept getting mixed up (with each other and with dragging). My computer may just be showing its age. I keep discovering new gestures by accident while trying to reproduce older ones 😆

**Comment by ratatatat12** [comment]
Oh thank you! This helps a lot
